pkg/conversions/virtualrouter_types_conversion_test.go

```diff
@@ -130,188 +131,6 @@ func TestConvert_CRD_WeightedTarget_To_SDK_WeightedTarget(t *testing.T) {
 	}
 }
 
```

Comment:
Is this test moved to the common file?

Reply:
I thought we discussed to restore this test?

Reply:
Checking

Reply:
The HeaderMatch method test case is moved to types_conversion_test .

Reply:
> Is this test moved to the common file?

Yes, The HeaderMatch method test case is moved to types_conversion_test .